From 40c981a7b39611ffe6570e37e2e9419c95114cf0 Mon Sep 17 00:00:00 2001
From: Antonio Larrosa <larrosa@kde.org>
Date: Wed, 11 Nov 2015 19:04:29 +0100
Subject: [PATCH 20/34] Adds a "count" property to UserModel

Adds a count property to UserModel so sddm themes can access the number of users in the system.
The name "count" is to mimic the "count" property in QML's standard ListModel.
---
 src/greeter/UserModel.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/greeter/UserModel.h b/src/greeter/UserModel.h
index 99d2770..e089b4d 100644
--- a/src/greeter/UserModel.h
+++ b/src/greeter/UserModel.h
@@ -32,6 +32,7 @@ namespace SDDM {
         Q_DISABLE_COPY(UserModel)
         Q_PROPERTY(int lastIndex READ lastIndex CONSTANT)
         Q_PROPERTY(QString lastUser READ lastUser CONSTANT)
+        Q_PROPERTY(int count READ rowCount CONSTANT)
     public:
         enum UserRoles {
             NameRole = Qt::UserRole + 1,
-- 
2.5.0

